docs/mkdocs.yml

```diff
@@ -97,67 +96,66 @@ nav:
-          - Reflection & Critique:
-              - Basic Reflection: tutorials/reflection/reflection.ipynb
-              - Reflexion: tutorials/reflexion/reflexion.ipynb
-              - Language Agent Tree Search: tutorials/lats/lats.ipynb
-              - Self-Discovering Agent: tutorials/self-discover/self-discover.ipynb
-          - Evaluation & Analysis:
-              - Chatbot Eval via Sim:
-                  - Agent-based: tutorials/chatbot-simulation-evaluation/agent-simulation-evaluation.ipynb
-                  - In LangSmith: tutorials/chatbot-simulation-evaluation/langsmith-agent-simulation-evaluation.ipynb
-          - Text Mining:
-              - TNT-LLM: tutorials/tnt-llm/tnt-llm.ipynb
-          - Web Navigation: tutorials/web-navigation/web_voyager.ipynb
-          - Competitive Programming: tutorials/usaco/usaco.ipynb
-          - SQL: tutorials/sql-agent.ipynb
+      - Chatbots:
+          - Customer Support: tutorials/customer-support/customer-support.ipynb
+          - Prompt Generation from User Requirements: tutorials/chatbots/information-gather-prompting.ipynb
+          - Code Assistant: tutorials/code_assistant/langgraph_code_assistant.ipynb
+      - RAG:
+          - Adaptive RAG: tutorials/rag/langgraph_adaptive_rag.ipynb
+          - Adaptive RAG using local LLMs: tutorials/rag/langgraph_adaptive_rag_local.ipynb
+          - Agentic RAG: tutorials/rag/langgraph_agentic_rag.ipynb
+          - Corrective RAG (CRAG): tutorials/rag/langgraph_crag.ipynb
+          - Corrective RAG (CRAG) using local LLMs: tutorials/rag/langgraph_crag_local.ipynb
+          - Self-RAG: tutorials/rag/langgraph_self_rag.ipynb
+          - Self-RAG using local LLMs: tutorials/rag/langgraph_self_rag_local.ipynb
+      - Agent Architectures:
+        - Multi-Agent Systems:
+            - Collaboration: tutorials/multi_agent/multi-agent-collaboration.ipynb
+            - Supervision: tutorials/multi_agent/agent_supervisor.ipynb
+            - Hierarchical Teams: tutorials/multi_agent/hierarchical_agent_teams.ipynb
+        - Planning Agents:
+            - Plan-and-Execute: tutorials/plan-and-execute/plan-and-execute.ipynb
+            - Reasoning without Observation: tutorials/rewoo/rewoo.ipynb
+            - LLMCompiler: tutorials/llm-compiler/LLMCompiler.ipynb
+        - Reflection & Critique:
+            - Basic Reflection: tutorials/reflection/reflection.ipynb
+            - Reflexion: tutorials/reflexion/reflexion.ipynb
+            - Language Agent Tree Search: tutorials/lats/lats.ipynb
+            - Self-Discover Agent: tutorials/self-discover/self-discover.ipynb
+      - Evaluation & Analysis:
+          - Chatbot Evaluation via Simulation:
+              - Agent-based: tutorials/chatbot-simulation-evaluation/agent-simulation-evaluation.ipynb
+              - In LangSmith: tutorials/chatbot-simulation-evaluation/langsmith-agent-simulation-evaluation.ipynb
+      - SQL: tutorials/sql-agent.ipynb
```

Comment:
SQL probably makes sense as a part of RAG right?

Reply:
yea was contemplating the same -- let me combine!